downstream/modules/platform/con-building-an-execution-environment-in-a-disconnected-environment.adoc

```diff
@@ -6,21 +6,34 @@
 = Building an {ExecEnvShort} in a disconnected environment
 
 
-link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/creating_and_consuming_execution_environments/index[Creating execution environments] for AAP is a common task which works differently in disconnected environments. When building a custom execution environment, the ansible-builder tool defaults to downloading content from the following locations on the internet:
+link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/creating_and_consuming_execution_environments/index[Creating execution environments] for {PlatformNameShort} is a common task which works differently in disconnected environments. When building a custom {ExecEnvShort}, the ansible-builder tool defaults to downloading content from the following locations on the internet:
 
-* Red Hat {HubNameStart} (console.redhat.com) or Ansible Galaxy (galaxy.ansible.com) for any Ansible content collections added to the execution environment image.
+* Red Hat {HubNameStart} ({Console}) or {Galaxy} (galaxy.ansible.com) for any Ansible content collections added to the {ExecEnvShort} image.
 
 * PyPI (pypi.org) for any python packages required as collection dependencies.
 
 * RPM repositories such as the RHEL or UBI repositories (cdn.redhat.com) for adding or updating RPMs to the execution environment image, if needed. 
 
 * registry.redhat.io for access to the base container images.
 
-Mirrored PyPI content once transferred into the disconnected network can be made available using a web server or an artifact repository like Nexus.  The RHEL and UBI repository content can be exported from an internet-facing Red Hat Satellite server, copied into the disconnected environment, then imported into a disconnected Satellite so it is available for building custom execution environments.  See link:https://access.redhat.com/documentation/en-us/red_hat_satellite/6.14/html-single/installing_satellite_server_in_a_disconnected_network_environment/index#iss_export_sync_in_an_air_gapped_scenario[ISS Export Sync in an Air-Gapped Scenario] for details.
+* Red Hat does not have an IP list for the container registry domains or cdn-ubi.redhat.com as for for cdn.redhat.com. 
```

Comment:
```suggestion
* Red Hat does not have an IP list for the container registry domains or cdn-ubi.redhat.com as for cdn.redhat.com. 
```
Does this mean that Red Hat does have an IP list for cdn.redhat.com but not for the first 2?

Reply:
I'm not sure. Hence the technical review. I've added a link to the KCS article the information came from, though that's a bit vague too.